### DISCUSSION TITLE
Width the calendar component panel in month selection

### DISCUSSION THREAD

**Discussion by LLloKoLaDKa** [primefaces/community#136] [PrimeReact] [answered]
![Screenshot_1](https://github.com/primefaces/community/assets/72572055/9e06d45d-a081-4fbd-9384-7176aade88d5)
The calendar component in month selection mode does not allow you to control the width of the pop-up selection panel (p-datepicker). Moreover, it is possible to control the width of the container if the full date selection mode is set.

Attempts to control width were made by specifying the "style" attribute. For some reason, when specifying this attribute, the width always becomes 127px. The width of the panel is taken from the parent (that is, the input)

Technologies: React + JS

**Comment by melloware** [comment, marked as answer]
@LLloKoLaDKa here is my reproducer: https://stackblitz.com/edit/react-mt3y5i?file=src%2FApp.js

Looks OK to me?

![image](https://github.com/primefaces/community/assets/4399574/3ca63ad1-8b40-4b3d-900c-1547e9b22b3e)
